main.go

```diff
@@ -5,32 +5,64 @@ import (
 	"log"
 	"net/http"
 	"os"
+	"strings"
+	"text/template"
 
 	"github.com/Clever/mesos-visualizer/ecs"
 )
 
 var (
-	Cluster            string
+	Clusters           map[string]string
 	AWSAccessKeyID     string
 	AWSSecretAccessKey string
 )
 
 func init() {
-	Cluster = getEnv("CLUSTER")
+	Clusters = getEnvJSON("CLUSTERS")
 	AWSAccessKeyID = getEnv("AWS_ACCESS_KEY_ID")
 	AWSSecretAccessKey = getEnv("AWS_SECRET_ACCESS_KEY")
 
 }
 
 func main() {
-	http.HandleFunc("/resources.json", resourcesHandler)
-	http.Handle("/", http.StripPrefix("/", http.FileServer(http.Dir("./static"))))
+	http.HandleFunc("/resources/", resourcesHandler)
+	http.HandleFunc("/", indexHandler)
+	http.Handle("/static/", http.StripPrefix("/static/", http.FileServer(http.Dir("./static"))))
 	log.Print("Listening on port 80...")
 	log.Fatal(http.ListenAndServe(":80", nil))
 }
 
-func resourcesHandler(w http.ResponseWriter, r *http.Request) {
-	c := ecs.NewClient(Cluster, AWSAccessKeyID, AWSSecretAccessKey)
+func indexHandler(w http.ResponseWriter, r *http.Request) {
+	tmpl, err := template.New("home").Parse(`
+	<!DOCTYPE html>
+	<head>
+		<meta charset="utf-8">
+		<title>ECS Visualizations</title>
+	</head>
+	<body>
+		{{ range $name, $arn := . }}
+		<h2>{{$name}}</h2>
+		<ul>
+			<li><a href="./static/sunburst.html?{{$arn}}">Resource Utilization - Sunburst</a></li>
```

Comment:
My preference here would be to use the name instead of the ARN, and perform a key lookup in `Clusters` in the resourcesHandler to prevent arbitrary ARNs.

Reply:
> ... look up ... arbitrary ARNs

I considered that a feature, not a bug 😉 

But I can change.

Reply:
Changed in 2230386048dbe432aad999c7a96f55bf429e3f3f